packages/client/src/stores/notification.js

```diff
@@ -1,8 +1,6 @@
 import { writable, get } from "svelte/store"
 import { routeStore } from "./routes"
 
-const NOTIFICATION_TIMEOUT = 3000
```

Comment:
I would keep this here and maybe just rename it to DEFAULT_NOTIFICATION_TIMEOUT. It's nice to have constants like this defined in an easy to find location, rather than having hardcoded values buried in files.